gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -16,30 +16,44 @@ description: Simple template to deploy a single compute instance
 
 parameters:
  key_name:
-  type: string
+   type: string
+   default: 
+ username:
+   type: string
+   default: syseleven
+ instance_name:
+   type: string
+   default: kickstart
```

Comment:
Really good idea! I like it a lot.

I would not parametrize this, but instead use `OS::stack_name` https://docs.openstack.org/heat/latest/template_guide/hot_spec.html#pseudo-parameters

Reply:
See commit 55e78290a0cba68dfb6b7c3248faa0cdf294eb92